accessibility/common-remediations.md

```diff
@@ -793,6 +794,29 @@ We _don't_ do this:
   <h3>Headline Topic 2</h3>
 ```
 
+### Missing heading
+
+#### What's the problem?
+
+A section or context lacks a heading. It may be a section on a page, or a standalone context like a modal dialog. 
+
+#### Who's affected by the problem?
+
+* Screen reader users
+* Users with cognitive disabilites or challenges
+
+#### Why's it a problem?
+
+Headings allow users to understand the structure of a page. When new sections or contexts are introduced without them, it can be difficult for users to understand how the content relates to the rest of the page. 
```

Comment:
```suggestion
Headings help users understand the structure of a page. When new sections or contexts are introduced without them, it can be difficult for users to understand how the content relates to the rest of the page. 
```